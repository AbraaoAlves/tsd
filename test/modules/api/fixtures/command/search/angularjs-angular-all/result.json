{
  "nameMatches": [
    {
      "path": "angularjs/angular-cookies.d.ts",
      "project": "angularjs",
      "name": "angular-cookies",
      "def": {
        "path": "angularjs/angular-cookies.d.ts",
        "commit": {
          "commitSha": "a8d8c2ac668fbb99cb60fb5d25c663e9356fee14"
        },
        "key": "angularjs/angular-cookies.d.ts-a8d8c2ac668fbb99cb60fb5d25c663e9356fee14",
        "solved": false,
        "dependencies": []
      },
      "history": []
    },
    {
      "path": "angularjs/angular-mocks.d.ts",
      "project": "angularjs",
      "name": "angular-mocks",
      "def": {
        "path": "angularjs/angular-mocks.d.ts",
        "commit": {
          "commitSha": "a8d8c2ac668fbb99cb60fb5d25c663e9356fee14"
        },
        "key": "angularjs/angular-mocks.d.ts-a8d8c2ac668fbb99cb60fb5d25c663e9356fee14",
        "solved": false,
        "dependencies": []
      },
      "history": []
    },
    {
      "path": "angularjs/angular-resource.d.ts",
      "project": "angularjs",
      "name": "angular-resource",
      "def": {
        "path": "angularjs/angular-resource.d.ts",
        "commit": {
          "commitSha": "a8d8c2ac668fbb99cb60fb5d25c663e9356fee14"
        },
        "key": "angularjs/angular-resource.d.ts-a8d8c2ac668fbb99cb60fb5d25c663e9356fee14",
        "solved": false,
        "dependencies": []
      },
      "history": []
    },
    {
      "path": "angularjs/angular-sanitize.d.ts",
      "project": "angularjs",
      "name": "angular-sanitize",
      "def": {
        "path": "angularjs/angular-sanitize.d.ts",
        "commit": {
          "commitSha": "a8d8c2ac668fbb99cb60fb5d25c663e9356fee14"
        },
        "key": "angularjs/angular-sanitize.d.ts-a8d8c2ac668fbb99cb60fb5d25c663e9356fee14",
        "solved": false,
        "dependencies": []
      },
      "history": []
    },
    {
      "path": "angularjs/angular-scenario.d.ts",
      "project": "angularjs",
      "name": "angular-scenario",
      "def": {
        "path": "angularjs/angular-scenario.d.ts",
        "commit": {
          "commitSha": "a8d8c2ac668fbb99cb60fb5d25c663e9356fee14"
        },
        "key": "angularjs/angular-scenario.d.ts-a8d8c2ac668fbb99cb60fb5d25c663e9356fee14",
        "solved": false,
        "dependencies": []
      },
      "history": []
    },
    {
      "path": "angularjs/angular.d.ts",
      "project": "angularjs",
      "name": "angular",
      "def": {
        "path": "angularjs/angular.d.ts",
        "commit": {
          "commitSha": "a8d8c2ac668fbb99cb60fb5d25c663e9356fee14"
        },
        "key": "angularjs/angular.d.ts-a8d8c2ac668fbb99cb60fb5d25c663e9356fee14",
        "solved": false,
        "dependencies": []
      },
      "history": []
    }
  ],
  "selection": [
    {
      "path": "angularjs/angular-cookies.d.ts",
      "commit": {
        "commitSha": "a8d8c2ac668fbb99cb60fb5d25c663e9356fee14"
      },
      "key": "angularjs/angular-cookies.d.ts-a8d8c2ac668fbb99cb60fb5d25c663e9356fee14",
      "solved": false,
      "dependencies": []
    },
    {
      "path": "angularjs/angular-mocks.d.ts",
      "commit": {
        "commitSha": "a8d8c2ac668fbb99cb60fb5d25c663e9356fee14"
      },
      "key": "angularjs/angular-mocks.d.ts-a8d8c2ac668fbb99cb60fb5d25c663e9356fee14",
      "solved": false,
      "dependencies": []
    },
    {
      "path": "angularjs/angular-resource.d.ts",
      "commit": {
        "commitSha": "a8d8c2ac668fbb99cb60fb5d25c663e9356fee14"
      },
      "key": "angularjs/angular-resource.d.ts-a8d8c2ac668fbb99cb60fb5d25c663e9356fee14",
      "solved": false,
      "dependencies": []
    },
    {
      "path": "angularjs/angular-sanitize.d.ts",
      "commit": {
        "commitSha": "a8d8c2ac668fbb99cb60fb5d25c663e9356fee14"
      },
      "key": "angularjs/angular-sanitize.d.ts-a8d8c2ac668fbb99cb60fb5d25c663e9356fee14",
      "solved": false,
      "dependencies": []
    },
    {
      "path": "angularjs/angular-scenario.d.ts",
      "commit": {
        "commitSha": "a8d8c2ac668fbb99cb60fb5d25c663e9356fee14"
      },
      "key": "angularjs/angular-scenario.d.ts-a8d8c2ac668fbb99cb60fb5d25c663e9356fee14",
      "solved": false,
      "dependencies": []
    },
    {
      "path": "angularjs/angular.d.ts",
      "commit": {
        "commitSha": "a8d8c2ac668fbb99cb60fb5d25c663e9356fee14"
      },
      "key": "angularjs/angular.d.ts-a8d8c2ac668fbb99cb60fb5d25c663e9356fee14",
      "solved": false,
      "dependencies": []
    }
  ],
  "written": {}
}