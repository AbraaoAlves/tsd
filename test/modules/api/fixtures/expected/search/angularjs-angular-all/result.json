{
  "selection": [
    {
      "path": "angularjs/angular-cookies.d.ts",
      "key": "angularjs/angular-cookies.d.ts-4cc9beb4057a46d0d81fdbf8621fa693b7157a0c",
      "solved": false,
      "commit": {
        "commitSha": "4cc9beb4057a46d0d81fdbf8621fa693b7157a0c"
      },
      "blob": {
        "sha": "60eebbacb02fad10c66df953fbd7c546adb4464f"
      },
      "dependencies": []
    },
    {
      "path": "angularjs/angular-mocks.d.ts",
      "key": "angularjs/angular-mocks.d.ts-4cc9beb4057a46d0d81fdbf8621fa693b7157a0c",
      "solved": false,
      "commit": {
        "commitSha": "4cc9beb4057a46d0d81fdbf8621fa693b7157a0c"
      },
      "blob": {
        "sha": "a607422dd44e906d7e883197c3e00ae28ba3f36d"
      },
      "dependencies": []
    },
    {
      "path": "angularjs/angular-resource.d.ts",
      "key": "angularjs/angular-resource.d.ts-4cc9beb4057a46d0d81fdbf8621fa693b7157a0c",
      "solved": false,
      "commit": {
        "commitSha": "4cc9beb4057a46d0d81fdbf8621fa693b7157a0c"
      },
      "blob": {
        "sha": "f74f3eb4486ee7a943d50eb74d6cfe428c035e8a"
      },
      "dependencies": []
    },
    {
      "path": "angularjs/angular-route.d.ts",
      "key": "angularjs/angular-route.d.ts-4cc9beb4057a46d0d81fdbf8621fa693b7157a0c",
      "solved": false,
      "commit": {
        "commitSha": "4cc9beb4057a46d0d81fdbf8621fa693b7157a0c"
      },
      "blob": {
        "sha": "b8d1db50b58d7665e97aa622920a8259e800383d"
      },
      "dependencies": []
    },
    {
      "path": "angularjs/angular-sanitize.d.ts",
      "key": "angularjs/angular-sanitize.d.ts-4cc9beb4057a46d0d81fdbf8621fa693b7157a0c",
      "solved": false,
      "commit": {
        "commitSha": "4cc9beb4057a46d0d81fdbf8621fa693b7157a0c"
      },
      "blob": {
        "sha": "a28bdb0d77cb62805903ca78cc7d02b2810e6e9f"
      },
      "dependencies": []
    },
    {
      "path": "angularjs/angular-scenario.d.ts",
      "key": "angularjs/angular-scenario.d.ts-4cc9beb4057a46d0d81fdbf8621fa693b7157a0c",
      "solved": false,
      "commit": {
        "commitSha": "4cc9beb4057a46d0d81fdbf8621fa693b7157a0c"
      },
      "blob": {
        "sha": "cdd66edf71ee2fec541108c747199ee7444b9917"
      },
      "dependencies": []
    },
    {
      "path": "angularjs/angular.d.ts",
      "key": "angularjs/angular.d.ts-4cc9beb4057a46d0d81fdbf8621fa693b7157a0c",
      "solved": false,
      "commit": {
        "commitSha": "4cc9beb4057a46d0d81fdbf8621fa693b7157a0c"
      },
      "blob": {
        "sha": "3b0599ac4dac134e917faaa55bf2bb8da3de7a3e"
      },
      "dependencies": []
    }
  ],
  "definitions": [
    {
      "path": "angularjs/angular-cookies.d.ts",
      "project": "angularjs",
      "name": "angular-cookies",
      "head": {
        "path": "angularjs/angular-cookies.d.ts",
        "key": "angularjs/angular-cookies.d.ts-4cc9beb4057a46d0d81fdbf8621fa693b7157a0c",
        "solved": false
      },
      "history": []
    },
    {
      "path": "angularjs/angular-mocks.d.ts",
      "project": "angularjs",
      "name": "angular-mocks",
      "head": {
        "path": "angularjs/angular-mocks.d.ts",
        "key": "angularjs/angular-mocks.d.ts-4cc9beb4057a46d0d81fdbf8621fa693b7157a0c",
        "solved": false
      },
      "history": []
    },
    {
      "path": "angularjs/angular-resource.d.ts",
      "project": "angularjs",
      "name": "angular-resource",
      "head": {
        "path": "angularjs/angular-resource.d.ts",
        "key": "angularjs/angular-resource.d.ts-4cc9beb4057a46d0d81fdbf8621fa693b7157a0c",
        "solved": false
      },
      "history": []
    },
    {
      "path": "angularjs/angular-route.d.ts",
      "project": "angularjs",
      "name": "angular-route",
      "head": {
        "path": "angularjs/angular-route.d.ts",
        "key": "angularjs/angular-route.d.ts-4cc9beb4057a46d0d81fdbf8621fa693b7157a0c",
        "solved": false
      },
      "history": []
    },
    {
      "path": "angularjs/angular-sanitize.d.ts",
      "project": "angularjs",
      "name": "angular-sanitize",
      "head": {
        "path": "angularjs/angular-sanitize.d.ts",
        "key": "angularjs/angular-sanitize.d.ts-4cc9beb4057a46d0d81fdbf8621fa693b7157a0c",
        "solved": false
      },
      "history": []
    },
    {
      "path": "angularjs/angular-scenario.d.ts",
      "project": "angularjs",
      "name": "angular-scenario",
      "head": {
        "path": "angularjs/angular-scenario.d.ts",
        "key": "angularjs/angular-scenario.d.ts-4cc9beb4057a46d0d81fdbf8621fa693b7157a0c",
        "solved": false
      },
      "history": []
    },
    {
      "path": "angularjs/angular.d.ts",
      "project": "angularjs",
      "name": "angular",
      "head": {
        "path": "angularjs/angular.d.ts",
        "key": "angularjs/angular.d.ts-4cc9beb4057a46d0d81fdbf8621fa693b7157a0c",
        "solved": false
      },
      "history": []
    }
  ]
}